snow/consensus/snowball/consensus_performance_test.go

```diff
@@ -26,29 +26,28 @@ func TestDualAlphaOptimization(t *testing.T) {
 			BetaVirtuous:    15,
 			BetaRogue:       20,
 		}
-		seed uint64 = 0
+		seed   uint64 = 0
+		source        = prng.NewMT19937()
```

Comment:
q: should we encapsulate this within a rand.DefaultSource getter?

Reply:
I don't think we really care too much about the actual randomness used... just that it is deterministic. I don't really seen the benefit in adding a helper here.